### PR TITLE
fix bug in opc ua driver response handling when fields are not found

### PR DESCRIPTION
**Pull request by stefah**
Fixes a bug within the check for returned fields in the opc ua driver connection response, where the status code was wrongly compared.
Updates the milo dependencies.

### CODE REVIEW

**Comment by chrisdutz** [comment]
Looks good to me ... and I did see it fail before and work with the change ;-)

File: plc4j/drivers/opcua/src/main/java/org/apache/plc4x/java/opcua/connection/OpcuaTcpPlcConnection.java [hunk 1]
```diff
@@ -346,7 +346,7 @@ public void unregister(PlcConsumerRegistration registration) {
             for(int counter = 0; counter < readValueIds.size(); counter++){
                 PlcResponseCode resultCode = PlcResponseCode.OK;
                 BaseDefaultFieldItem stringItem = null;
-                if(readValues == null || readValues.size() <= counter || readValues.get(counter).getStatusCode() != StatusCode.GOOD){
+                if(readValues == null || counter > readValues.size() || !readValues.get(counter).getStatusCode().isGood()){
```
Comment (nalim2) [inline plc4j/drivers/opcua/src/main/java/org/apache/plc4x/java/opcua/connection/OpcuaTcpPlcConnection.java]: Correct but it has to be `counter >= readValues.size()` or?

Comment (chrisdutz) [reply plc4j/drivers/opcua/src/main/java/org/apache/plc4x/java/opcua/connection/OpcuaTcpPlcConnection.java]: Actually I think you are right as if it's equal to size it must produce an array index out of bounds exception. Stefan, could you please re evaluate that?

Comment (stefah) [reply plc4j/drivers/opcua/src/main/java/org/apache/plc4x/java/opcua/connection/OpcuaTcpPlcConnection.java]: you are correct, will fix that.. :)

**Pull request merged**